scripts/install-worker.sh

```diff
@@ -473,7 +474,7 @@ fi
 ### SSM Agent ##################################################################
 ################################################################################
 
-sudo yum install -y amazon-ssm-agent
+sudo yum install -y https://s3.${BINARY_BUCKET_REGION}.${S3_DOMAIN}/amazon-ssm-${BINARY_BUCKET_REGION}/${SSM_AGENT_VERSION}/linux_${ARCH}/amazon-ssm-agent.rpm
```

Comment:
As we did with the AWS CLI, we need to fall back to installing from the `yum` repos in isolated regions: https://github.com/awslabs/amazon-eks-ami/blob/master/scripts/install-worker.sh#L122-L139

Reply:
done